build.ps1

```diff
@@ -0,0 +1,30 @@
+param (

```

Comment:
This shouldn't be needed with the new dotnet tooling

Reply:
Take a look this:

https://search.d2l.dev/xref/Brightspace/D2L.InfluxDB.LineProtocol.Reader/.github/workflows/build.yml?r=f3b8adf1

My build scripts pretty much all look like this now.  I could wrap this up in a reusable action